aspnetcore/fundamentals/http-requests.md

```diff
@@ -0,0 +1,394 @@
+---
+title: Initiate HTTP requests
+author: stevejgordon
+description: Learn about using the HttpClientFactory to managed logic HttpClient instances.
+manager: wpickett
+ms.author: 
+ms.custom: mvc
+ms.date: 02/21/2018
+ms.prod: asp.net-core
+ms.technology: aspnet
+ms.topic: article
+uid: fundamentals/http-requests
+---
+# Initiate HTTP requests
+
+By [Glenn Condron](https://github.com/glennc), [Ryan Nowak](https://github.com/rynowak), and [Steve Gordon](https://github.com/stevejgordon) 
+
+A `HttpClientFactory` can be registered and used to configure and consume `HttpClient` instances in an app. It offers the following benefits:
```

Comment:
Should be `IHttpClientFactory` not `HttpClientFactory`

> to configure and consume

Can we say create instead of consume? The factory doesn't *do* HTTP it just sets up the clients for you.